docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -60,23 +73,53 @@ As the popularity of Android development grows, the need for efficient build aut
 
 **Implemented Features:**
 
-- Probe build the project to find if default java home configuration is compatible.
-- In case of incompatibility try to find a compatible java home in the given order:
+- Probe build the project to find if default Java Home configuration is compatible.
+- In case of incompatibility try to find a compatible Java Home in the given order:
     1. GradleJavaHome (Gradle Properties)
     2. UserJavaHome (Retrieved from preferences)
-- Notify client if project's default java home was incompatible and we switched to a different JavaHome for compatibility.
-- Notify client if the fallback logic couldn't find a compatible java home.
+- Notify client if project's default Java Home was incompatible and we switched to a different Java Home for compatibility.
+- Notify client if the fallback logic couldn't find a compatible Java Home.
+
+**Supporting Diagrams:**
+
+<div class="img_container">
+  <img src="./images/JavaHomeHandling.png" alt="Java Home Handling">
+</div>
 
 ### Android Support
 
-**Status:** Ongoing (Under Development)
+**Status:** Complete (Under Review)
 
 **Pull Request:** [#173](https://github.com/microsoft/build-server-for-gradle/pull/173)
 
 **Objectives:**
 
-- Correct classpath resolution.
-- Test execution.
+- Providing build targets for Android Java projects to the client
+
+**Pre-requisites:**
+
+- Understanding the entire Android build process and changes to the Android Gradle Plugin.
+
+**Implemented Features:**
+
+- Extracting Build Variants and their properties from Android projects via reflection.
+- Populating a GradleSourceSet (build target model) with build variants properties.
+- Providing Android SDK components and R file via dependency modules of the build target.
+
+**Limitations:**
+
+- The current implementation only allows for recognition of Java sources.
+- `R.jar` file is provided via ModuleDependency but it may only work for Android Applcation projects.
+- If `ANDROID_HOME` is not configured there is no fallback implementation, the Android Components will simply not be provided.
+
+**Supporting Diagrams:**
+
+<div class="img_container">
```

Comment:
This should not be really required, a normal Markdown image inclusion would be enough